server/routes/contactPersonMigration/contactPersonProfileDetailsMigration/contactPersonProfileDetailsMigrationController.ts

```diff
@@ -0,0 +1,188 @@
+import { Request, Response } from 'express'
+import moment from 'moment'
+import { PrisonerFilteredMigrationForm } from 'express-session'
+import NomisMigrationService from '../../../services/contactperson/profiledetails/contactPersonProfileDetailsNomisMigrationService'
+import { Context } from '../../../services/nomisMigrationService'
+import { MigrationHistory } from '../../../@types/migration'
+import { buildUrl } from '../../../utils/applicationInsightsUrlBuilder'
+import trimForm from '../../../utils/trim'
+import logger from '../../../../logger'
+import NomisPrisonerService from '../../../services/contactperson/profiledetails/contactPersonProfileDetailsNomisPrisonerService'
+
+interface Filter {
+  prisonerNumber?: string
+}
+
+function context(res: Response): Context {
+  return {
+    username: res?.locals?.user?.username,
+    token: res?.locals?.user?.token,
+  }
+}
+
+export default class ContactPersonProfileDetailsMigrationController {
```

Comment:
I've tested locally but not put the integration tests in yet. Also planning to tweak the app insights links as the entity migration ignored scenario is not relevant to us.